core/src/main/java/com/linecorp/armeria/common/metric/NettyMetrics.java

```diff
@@ -0,0 +1,133 @@
+/*
+ *  Copyright 2023 LINE Corporation
+ *
+ *  LINE Corporation licenses this file to you under the Apache License,
+ *  version 2.0 (the "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at:
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package com.linecorp.armeria.common.metric;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.TimeUnit;
+
+import com.linecorp.armeria.common.util.Ticker;
+import com.linecorp.armeria.internal.common.metric.MicrometerUtil;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.binder.MeterBinder;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.SingleThreadEventLoop;
+import io.netty.util.concurrent.EventExecutor;
+
+/**
+ * A {@link MeterBinder} to observe Netty's runtime. The following stats are currently exported per
+ * registered {@link MeterIdPrefix}.
+ *
+ * <p>
+ *  - "event.loops.num.workers" (gauge) - the total number of Netty's event loops
+ *  - "event.loops.pending.tasks" (gauge) - the total number of IO tasks waiting to be run on event loops*
+ * </p>
+ */
+public final class NettyMetrics implements MeterBinder {
+
+    private MeterIdPrefix idPrefix;
+    private EventLoopGroup eventLoopGroup;
+
+    /**
+     * Creates an instance of {@link NettyMetrics}.
+     */
+    public NettyMetrics(MeterIdPrefix idPrefix, EventLoopGroup eventLoopGroup) {
+        this.idPrefix = requireNonNull(idPrefix, "idPrefix");
+        this.eventLoopGroup = requireNonNull(eventLoopGroup, "eventLoopGroup");
+    }
+
+    @Override
+    public void bindTo(MeterRegistry registry) {
+        final Self metrics = MicrometerUtil.register(registry, idPrefix, Self.class, Self::new);
+        metrics.add(eventLoopGroup);
+    }
+
+    private static final long UPDATE_INTERVAL_NANOS = TimeUnit.SECONDS.toNanos(3);
```

Comment:
Code style: Should we move `UPDATE_INTERVAL_NANOS` above the member fields?
https://armeria.dev/community/developer-guide#organize